agent/ecscni/plugin.go

```diff
@@ -135,6 +162,48 @@ func (client *cniClient) constructNetworkConfig(cfg *Config) (*libcni.NetworkCon
 		IPAM:       ipamConf,
 	}
 
+	bridgeConfBytes, err := json.Marshal(bridgeConf)
+	if err != nil {
+		seelog.Errorf("Marshal bridge configuration failed, error: %v", err)
+		return nil, err
+	}
+	networkConfig := &libcni.NetworkConfig{
+		Network: &types.NetConf{
+			Type: ECSBridgePluginName,
+		},
+		Bytes: bridgeConfBytes,
+	}
+
+	return networkConfig, nil
+}
+
+func (client *cniClient) createBridgeConfig(cfg *Config) (*libcni.NetworkConfig, error) {
```

Comment:
It looks like there's a bunch of duplicate code between `createBridgeConfig` and `createBridgeConfigWithIPAM`.  Can we share instead of duplicating?